python/paddle/trainer_config_helpers/tests/configs/test_config_parser_for_non_file_config.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# Copyright (c) 2016 Baidu, Inc. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import getopt
+
+whole = False
+opts, args = getopt.getopt(sys.argv[1:], "", ["whole"])
+for op, value in opts:
+  if op == "--whole":
+    whole = True
+
+cmdstr = """
+from paddle.trainer.config_parser import *
+from paddle.trainer_config_helpers import *
```

Comment:
This implementation is too tricky. Maybe the simplest implementation is just to write a single file to test `parse_config` from a python function with a very simple neural network.  Although the test-coverage is not full we could test this mechanism work well or not.